src/aiida/cmdline/commands/cmd_profile.py

```diff
@@ -136,17 +136,29 @@ def profile_setup():
 @setup.SETUP_BROKER_HOST()
 @setup.SETUP_BROKER_PORT()
 @setup.SETUP_BROKER_VIRTUAL_HOST()
-@options.NON_INTERACTIVE()
+@options.INTERACTIVE()
 @click.pass_context
 def profile_configure_rabbitmq(ctx, profile, **kwargs):
     """Configure RabbitMQ for a profile.
 
     Enable RabbitMQ for a profile that was created without a broker, or reconfigure existing connection details.
     """
+    from aiida.brokers.rabbitmq.defaults import detect_rabbitmq_config
+
     profile.set_process_controller(name='core.rabbitmq', config=kwargs)
     ctx.obj.config.update_profile(profile)
     ctx.obj.config.store()
 
+    broker_config = detect_rabbitmq_config(**kwargs)
+
+    if broker_config is None:
+        connection_params = {key: value for key, value in kwargs.items() if key.startswith('broker_')}
+        echo.echo_warning(
```

Comment:
I was thinking that maybe we would want to actually prompt and ask if they want to apply the changes. This would be useful in case they made a typo in the options and they can cancel. But that would also require adding a force option to skip this check for when it is used in scripts